unime/src/routes/credentials/[id]/OpenBadgeRenderer.svelte

```diff
@@ -24,14 +24,28 @@
 
   {#if credential.data.credentialSubject?.achievement?.criteria?.narrative}
     <div class="prose prose-sm rounded-xl bg-background p-4 dark:prose-invert">
-      <h2>{$LL.CREDENTIAL.DETAILS.CONTENTS()}</h2>
+      <h2>{$LL.CREDENTIAL.DETAILS.OPEN_BADGES.CRITERIA()}</h2>
```

Comment:
Just for me to understand, do the changes in this file mean that previously we just showed the whole contents of a badge, and now we **only** show the criteria? Is that what we want? Should we perhaps keep the `CONTENTS` the way it was and then in addition to that add the Criteria the way you did?